osquery/events/linux/auditdnetlink.cpp

```diff
@@ -668,6 +675,149 @@ bool AuditdNetlink::configureAuditService() noexcept {
   return true;
 }
 
+bool AuditdNetlink::clearAuditConfiguration() noexcept {
+  int seq = audit_request_rules_list_data(audit_netlink_handle_);
+  if (seq <= 0) {
+    VLOG(1) << "Failed to list the audit rules";
+    return false;
+  }
+
+  // Attempt to list all rules
+  std::vector<AuditRuleDataObject> rule_object_list;
+
+  for (int read_retry = 0; read_retry < 3; ++read_retry) {
+    bool netlink_ready = false;
+
+    for (int poll_retry = 0; poll_retry < 3; ++poll_retry) {
+      pollfd fds[] = {{audit_netlink_handle_, POLLIN, 0}};
+
+      errno = 0;
+      int poll_status = ::poll(fds, 1, 4);
+      if (poll_status == 0) {
+        continue;
+      }
+
+      if (poll_status < 0) {
+        VLOG(1) << "pool() failed with error " << errno;
```

Comment:
nit, spelling

Reply:
This is not a spelling error, but if you want I can change the **error** in the string into **errno**